graf2d/win32gdk/src/TGWin32.cxx

```diff
@@ -4802,6 +4801,11 @@ Window_t TGWin32::CreateWindow(Window_t parent, Int_t x, Int_t y,
    GdkColor background_color;
    ULong_t xmask = 0;
 
+   /* copy these bits from TGFrame.h, otherwise one should link against gui lib */
```

Comment:
Can you add a note in TGFrame.h that these constants in TGWin32.cxx need to be kept in sync?